main.go

```diff
@@ -21,29 +21,33 @@ var (
 	date        string
 	buildSource = "unknown"
 
-	configFlag    = flag.Bool("config", false, "Print the current default config")
-	debuggingFlag = flag.Bool("debug", false, "a boolean")
-	versionFlag   = flag.Bool("v", false, "Print the current version")
+	configFlag    = false
+	debuggingFlag = false
 )
 
 func main() {
-	flag.Parse()
-	if *versionFlag {
-		fmt.Printf("commit=%s, build date=%s, build source=%s, version=%s, os=%s, arch=%s\n", commit, date, buildSource, version, runtime.GOOS, runtime.GOARCH)
-		os.Exit(0)
-	}
 
-	if *configFlag {
+	flaggy.SetName("lazydocker")
+	flaggy.SetDescription("The lazier way to manage everything docker")
+	flaggy.DefaultParser.AdditionalHelpPrepend = "https://github.com/jesseduffield/lazydocker"
+
+	flaggy.Bool(&configFlag, "c", "config", "Print the current default config")
+	flaggy.Bool(&debuggingFlag, "d", "debug", "a boolean")
+	flaggy.SetVersion(fmt.Sprintf("commit=%s, build date=%s, build source=%s, version=%s, os=%s, arch=%s\n", commit, date, buildSource, version, runtime.GOOS, runtime.GOARCH))
+
+	flaggy.Parse()
+
+	if configFlag {
 		var buf bytes.Buffer
 		yaml.NewEncoder(&buf).Encode(config.GetDefaultConfig())
 		fmt.Printf("%v\n", buf.String())
 		os.Exit(0)
 	}
 
 	// for now we're always in debug mode so we're not passing *debuggingFlag
-	*debuggingFlag = true
+	debuggingFlag = true
```

Comment:
we can actually remove lines 47 and 48 because that was only there for my own personal convenience :P

Reply:
Good shout! :)